_infra/helm/locust/locustfiles/requests.json

```diff
@@ -38,7 +38,7 @@
             "data": {
                 "body": "test secure message"
             },
-            "expected_response_text": "Message sent",
+            "expected_response_text": "Click on the survey name to complete your questionnaire",
```

Comment:
The "Message sent" appears in the flash message component in the page, but for some reason locally this doesn't appear. Therefore I've just asserted on the main survey page text and not the flash message. Need to work out why that's happening.